algebra/csc_tools/csc_utils.c

```diff
@@ -1,5 +1,32 @@
-#include "cs.h"
 
+#ifndef EMBEDDED
```

Comment:
I agree with this, but I did not change any of the EMBEDDED conditional compilation logic in the code where possible.   I think it would be easier to rework the handling of the EMBEDDED option via cmake across the whole of the code base as a separate undertaking.

Reply:
Noted as a v1.0 update [here](https://github.com/oxfordcontrol/osqp/projects/1#column-1967224).